controllers/C_DocumentCategory.class.php

```diff
@@ -93,24 +118,6 @@ function delete_node_action_process($id) {
 		return $this->list_action();
 	}
 
-	function edit_action_process() {
-		if ($_POST['process'] != "true")
-			return;
-		//print_r($_POST);
-		if (is_numeric($_POST['id'])) {
-			$this->document_categories[0] = new Pharmacy($_POST['id']);
```

Comment:
Yeah my guess is this function was left over from cloning pharmacy code in initial development.